### ISSUE TITLE
README.md - update to highlight "DO **NOT** USE WITH IPYTHON 7" - it WILL corrupt your installation. 

### ISSUE THREAD

**Issue by WaterSibilantFalling** [ipython/rlipython#22]
A single sentence at the top of the README.md, should be a quick fix

**Comment by Carreau** [comment]
I think corrupting is a strong word. If you disable and remove rlipython
IPython has no reasons to be broken.

If you send a pr updating the readme I'd be happy to merge. Updating
setup.py to mark the dependencies as IPython<7 is also a possibility. And
actually updating this repo to work with 7+ would be nice. It's just an
open source project so we usually scratch our own itch on our own free
time. Contributions and help always welcome.

**Comment by WaterSibilantFalling** [comment]
> I think corrupting is a strong word. If you disable and remove rlipython
IPython has no reasons to be broken.

Well, fyi, after trying to remove rlipython, I can't run run ipython

``` 
prompt > ipython
=========
 IPython
=========

Tools for Interactive Computing in Python
=========================================
      :
      :    
      :

--cache-size=<Int> (InteractiveShell.cache_size)
    Default: 1000
    Set the size of the output cache.  The default is 1000, you can change it
    permanently in your config file.  Setting it to 0 completely disables the
    caching system, and the minimum value accepted is 3 (if you provide a value
    less than 3, it is reset to 0 and a warning is issued).  This limit is
    defined because otherwise you'll spend more time re-flushing a too small
    cache than working**[TerminalIPythonApp] CRITICAL | Bad config encountered during initialization:
[TerminalIPythonApp] CRITICAL | The 'interactive_shell_class' trait of <IPython.terminal.ipapp.TerminalIPythonApp object at 0x000001E7AB770048> instance must be a type, but 'rlipython.TerminalInteractiveShell' could not be imported**


To see all available configurables, use `--help-all`

```

Note:

*[TerminalIPythonApp] CRITICAL | Bad config encountered during initialization:
[TerminalIPythonApp] CRITICAL | The 'interactive_shell_class' trait of <IPython.terminal.ipapp.TerminalIPythonApp object at 0x000001E7AB770048> instance must be a type, but 'rlipython.TerminalInteractiveShell' could not be imported
*

You're right, I should edit setup.py and make a PR.

**Comment by baogorek** [comment]
Yeah I'm having the same problem. Can't get into ipython to uninstall, not even with --simple-prompt. Pip uninstall and reinstall of ipython doesn't fix the issue.

**Update**: Back on track here. I ended up creating a new ipython profile and logging into that to uninstall rlipython. You can also replace contents of .ipython\profile_default\ipython_config.json with {}.
